ilc/client/handlePageTransaction.js

```diff
@@ -78,19 +78,31 @@ const renderFakeSlot = slotName => {
     hiddenSlots.push(targetNode);
 };
 
-/**
- * @param {string} slotName
- * @param {string} willBe - possible values: rendered, removed, rerendered
- */
-export default function (slotName, willBe) {
-    if (!slotName || !willBe) return;
+export const slotCanBe = {
+    rendered: 'rendered',
+    removed: 'removed',
+    rerendered: 'rerendered',
+};
+
+export default function handlePageTransaction(slotName, slotWillBe) {
+    if (!slotName || !slotWillBe) return;
 
-    if (willBe === 'rendered') {
-        addContentListener(slotName);
-    } else if (willBe === 'removed') {
-        renderFakeSlot(slotName);
-    } else if (willBe === 'rerendered') {
-        renderFakeSlot(slotName);
-        addContentListener(slotName);
+    switch (slotWillBe) {
+        case slotCanBe.rendered: {
```

Comment:
I'm pretty sure that you don't need `{ ... }` brackets for case statements. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch

Reply:
Yes, it is not necessary
It is necessary only when you have a let or const variables that you may declare inside of brackets there